helpers.go

```diff
@@ -215,32 +214,17 @@ func preventImageOverwrite(repo, tag string) error {
 	return nil
 }
 
+// Expects the name of an image and returns it's ID.
 func getImageID(name string) (string, error) {
 	client := getDockerClient()
+	img, _, err := client.ImageInspectWithRaw(context.Background(), name)
 
-	repo, tag, err := parseImageName(name)
 	if err != nil {
-		return "", err
-	}
-	if tag == "latest" && !strings.Contains(name, tag) {
-		name = name + ":" + tag
-	}
-
-	images, err := client.ImageList(context.Background(), types.ImageListOptions{All: false, Filters: filters.NewArgs(filters.Arg("reference", repo))})
-	if err != nil {
-		return "", err
-	}
-
-	if len(images) == 0 {
 		return "", fmt.Errorf("Cannot find image %s", name)
 	}
-	for _, image := range images {
-		if arrayIncludeString(image.RepoTags, name) {
-			return image.ID, nil
-		}
-	}
+	imageID := img.ID
 
-	return "", fmt.Errorf("Cannot find image %s", name)
+	return imageID, err
```

Comment:
Although `err == nil`, please use `nil` in the return statement instead. It's easier to understand and a common pattern.